spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,179 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> empty/input.scss
+a {b: split("")}
```

Comment:
I think passing only one argument is expected to be an error. Should this by `split("", "")`?

Reply:
Fixed. Changed this to both `$string` and `$separator` being empty and returning an empty array.